# TransitionMsg.msg
std_msgs/MultiArrayLayout layout
float32[] state
float32[] action
float32[] reward
float32[] next_state